src/json2typescript/json-convert.ts

```diff
@@ -338,13 +338,13 @@ export class JsonConvert {
             console.log("----------");
         }
 
-        return jsonObject;
+        return JSON.parse(JSON.stringify(jsonObject));
```

Comment:
I suggest we use this trick to make sure a plain object is returned. This actually would finally close #87.

Reply:
Is that necessary?  From the code, since it's created just as `{}`, it looks like it should be returning a plain object.  When I ran a quick test, the constructor of the result came back as an Object, not as a class constructor.  Not sure if there's some other issue I'm missing?

Reply:
> Is that necessary?  From the code, since it's created just as `{}`, it looks like it should be returning a plain object.  When I ran a quick test, the constructor of the result came back as an Object, not as a class constructor.  Not sure if there's some other issue I'm missing?


I think it depends on the parameter you are passing. If I recall correcty, the serializaion sometimes keeps the original instance and just reassigns values. I will check later if that is true.

Reply:
I think you are right, it was unnecessary.